app/templates/components/insights-tabs.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each tabs as |tab|}}
+  <li class="insights-tab {{if (eq tab.slug selectedTab) 'active'}}">
+    <a data-tab={{tab.slug}} {{action 'setInsightTab' tab.slug}}>

```

Comment:
Would it make sense to bind tabs to query params? In this case you would be able to make the link with 
```handlebars
{{#link-to (query-params interval=tab.slug) }}
  {{tab.title}}
{{/link-to}}
```
Here's an example: https://github.com/travis-ci/travis-web/blob/master/app/templates/components/profile-nav.hbs#L57-L93

Also, it's recommended to use closure actions instead of old bubbling ones: 
```javascript
onclick={{action 'setInsightTab' tab.slug}}`
```

Reply:
Ooh yes, binding to query params makes sense to me, I think I'll switch to that.

Reply:
Oh and good to know about closure actions 👍 Thanks!